GitCommands/Git/GitModule.cs

```diff
@@ -1354,6 +1354,22 @@ public string ResetFile(string file)
             });
         }
 
+        public string ResetFiles(IReadOnlyList<string> files)
+        {
+            if (files == null || files.Count == 0)
+            {
+                return string.Empty;
+            }
+
+            return _gitExecutable.GetBatchOutput(new GitArgumentBuilder("checkout-index")
+                {
+                    "--index",
+                    "--force",
+                    "--"
+                }
+                .BuildBatchArguments(files.Select(item => item.ToPosixPath().Quote())));
+        }
```

Comment:
ditto

Reply:
What is ditto?

Reply:
Oh, add the test here as well right? OK

Reply:
> "ditto" - The same thing again (used in lists and accounts and often indicated by a ditto mark under the word or figure to be repeated).
https://www.lexico.com/en/definition/ditto

Basically, the same issue as pointed above

Reply:
Yes, I'd like to see tests for the new functionality

Reply:
Thought you were into Pokemon at first (same here). Test cases added.

Reply:
> Thought you were into Pokemon at first

LOL, I had to look up what Pokemons were :)